src/lib/C4Random.h

```diff
@@ -20,7 +20,7 @@
 #ifndef INC_C4Random
 #define INC_C4Random
 
-#include <inttypes.h>
+#include <cinttypes>
```

Comment:
I think this is technically not equivalent (like in src/c4group/CStdFile.h); but should be fine in practice.